libControls/ToolTip.cpp

```diff
@@ -102,6 +117,21 @@ void ToolTip::draw() const
 		auto& renderer = NAS2D::Utility<NAS2D::Renderer>::get();
 		renderer.drawBoxFilled(rect(), NAS2D::Color::DarkGray);
 		renderer.drawBox(rect(), NAS2D::Color::Black);
-		renderer.drawText(mFont, mFocusedControl->second, position() + PaddingSize);
+
+		auto linePosition = position() + PaddingSize;
+		processLines(mFocusedControl->second, [&renderer, this, &linePosition](const std::string& line) {
+			renderer.drawText(mFont, line, linePosition);
+			linePosition.y += mFont.height();
+		});
+	}
+}
+
+
+void ToolTip::processLines(const std::string& text, const std::function<void(const std::string&)>& lineProcessor) const
+{
+	std::istringstream stream(text);
+	for (std::string line; std::getline(stream, line);)
+	{
+		lineProcessor(line);
 	}
 }

```

Comment:
This function doesn't make use of any member fields of `ToolTip`, so also shouldn't be a regular member. A `static` member maybe, but perhaps it's best to remove this function from the class entirely, and put it in an unnamed namespace at the start of the .cpp file. The unnamed namespace will give it internal linkage, so it will never conflict with global names from other files during the link stage.